lib/mistral_ai_llm_controller_example/prompt.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:mistral_ai_chat_example_app/mistral_ai_llm_controller_example/model.dart';
+
+String controllerDescription(ControllerSettings currentSettings) =>
+    'You are a helpful assistant designed to output only JSON. '
+    'You are given command and a set of functions that you can perform. '
+    'Based on the command you should chose one of the provided functions '
+    'If you do not understand the command, you should return a following JSON. '
+    '{ "name": "unknown", "parameters": "" } '
+    'You can perform one function at a time. '
+    'Do not suggest new functions. '
+    'Do not return anything else than JSON. '
+    'Do not explain anything. '
+    'Current settings: '
+    '{ "color": ${currentSettings.color}, '
+    '"temperature": ${currentSettings.temperature}, '
+    '"volume": ${currentSettings.volume}}';
+
+const String controllerExample = '''
```

Comment:
I would name it little differently. For me it's more like controller instructions or controller functionality.

Reply:
I agree the name can be better.
I am thinking about making it more reusable by providing functions and examples as a parameter like a list of `ControllerFunction`
```
class ControllerFunction {
  ControllerFunction({
    required this.definition,
    required this.example,
  });
  String definition;
  String example;
}
```
WDYT? @gawi151